CMSgov/design-system
# [WNMGDS-254] Move the aria-live attributes to checkedChildren element

### Changed
- Move the aria-live attributes to checkedChildren element
- Reference https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions

### How to test
- Run the design system site locally and inspect the html elements to ensure that aria-live attributes is shown for checked children element only.

### Fixed
Fixes #484
